src/lib/proof-system/zkprogram.ts

```diff
@@ -614,6 +618,7 @@ async function compileProgram({
   forceRecompile: boolean;
   overrideWrapDomain?: 0 | 1 | 2;
 }) {
+  await initializeBindings();
```

Comment:
awaiting bindings in `compile()`

note: we don't need it in any of the provers because you have to call `compile()` before running them